tests/VersionRouteTest.php

```diff
@@ -289,4 +289,37 @@ public function testVersion(string $version, bool $isCorrectVersion, string $rou
 
         $response->assertStatus($status);
     }
+
+    public function testWithoutApiVersion(): void
+    {
+        $mock = $this
+            ->getMockPackageTestCaseCall()
+            ->withoutAPIVersion();
+
+        $this->assertRouteCalled($mock, '/test');
+
+        $mock->json('get', '/test');
+    }
+
+    public function testRouteWithSetApiVersion(): void
+    {
+        $mock = $this
+            ->getMockPackageTestCaseCall()
+            ->setAPIVersion(VersionEnum::V1);
+
+        $this->assertRouteCalled($mock, '/v1/test/');
```

Comment:
let's implement only one mock method and assert that the `call` method was called with expected argument

 ```suggestion
        $mock = $this->mockTestCaseExpectCallMethod('/v1/test/');

        $mock->setAPIVersion(VersionEnum::V1)->json('get', '/test/');
```